spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/source/BaseBatchReader.java

```diff
@@ -52,6 +55,7 @@
 
   protected CloseableIterable<ColumnarBatch> newBatchIterable(
       InputFile inputFile,
+      ByteBuffer keyMetadata,
```

Comment:
`ByteBuffer` is not used for key metadata, right? It's used for AADs and encryption keys.